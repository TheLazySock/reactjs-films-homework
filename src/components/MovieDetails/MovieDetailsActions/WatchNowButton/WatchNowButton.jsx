import React from 'react';
import style from './WatchNowButton.scss';

const WatchNowButton = () => (
  <button type="button" className={style.watchVideoBtn}>Watch Now</button>
);

export default WatchNowButton;
