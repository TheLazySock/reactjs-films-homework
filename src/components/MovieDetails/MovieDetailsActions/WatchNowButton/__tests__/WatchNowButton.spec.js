import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import WatchNowButton from '../index';

describe('WatchNowButton', () => {
  it('renders properly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<WatchNowButton />);
    const output = renderer.getRenderOutput();
    expect(output.type).toBe('button');
    expect(output).toMatchSnapshot();
  });
});
