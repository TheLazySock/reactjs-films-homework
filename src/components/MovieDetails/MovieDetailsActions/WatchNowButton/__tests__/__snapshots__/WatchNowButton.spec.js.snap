// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WatchNowButton renders properly 1`] = `
<button
  className="watchVideoBtn"
  type="button"
>
  Watch Now
</button>
`;
